content/docs/manuals/extending-oakestra/custom_resources.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Create a Custom Resource"
+summary: ""
+draft: false
+weight: 5
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Introduction to Custom Resources
+
+Custom Resources in Oakestra provide a mechanism to extend the system’s API dynamically by introducing new resource types. Inspired by Kubernetes’ Custom Resources, this feature allows developers to define and manage tailored data structures to meet specific needs, enabling Oakestra to adapt to a wide range of scenarios.
+
+This section demonstrates how to create, manage, and utilize Custom Resources within Oakestra.
+
+Custom Resources in Oakestra enable:
+1. **Dynamic API Extensions**: Add new resource types to the system beyond the default ones.
+2. **Declarative Management**: Store and retrieve structured data through API endpoints.
+3. **Custom Logic**: Use controllers to act on the data defined by the Custom Resources, enabling automation and state management.
+
```

Comment:
Also a "tip callout" with title "why should you use custom resources in Oakestra?"